### PR TITLE
(GH-1) Allow for large package sizes

### PR DESCRIPTION
**Pull request by ferventcoder**
- Set maxRequestLength to 2GB (2,147,482,548) based on 
  https://msdn.microsoft.com/en-us/library/e1f13641(v=vs.100).aspx
- Set maxAllowedContentLength to 2GB (2,147,482,548) based on
  https://msdn.microsoft.com/en-us/library/ms689462(v=vs.90).aspx

### CODE REVIEW

File: src/SimpleChocolateyServer/Web.config [hunk 1]
```diff
@@ -19,9 +19,14 @@
       <add name="ErrorMail" type="Elmah.ErrorMailModule, Elmah"/>
       <add name="ErrorFilter" type="Elmah.ErrorFilterModule, Elmah"/>
     </httpModules>
-    <httpRuntime maxRequestLength="31457280"/>
+    <httpRuntime maxRequestLength="2147482548"/>
   </system.web>
   <system.webServer>
+    <security>
+      <requestFiltering>
+        <requestLimits maxAllowedContentLength="2147482548" />
```
Comment (ferventcoder) [inline src/SimpleChocolateyServer/Web.config]: This is definitely Bytes, and it defaults to 28MB.

File: src/SimpleChocolateyServer/Web.config [hunk 2]
```diff
@@ -19,9 +19,14 @@
       <add name="ErrorMail" type="Elmah.ErrorMailModule, Elmah"/>
       <add name="ErrorFilter" type="Elmah.ErrorFilterModule, Elmah"/>
     </httpModules>
-    <httpRuntime maxRequestLength="31457280"/>
+    <httpRuntime maxRequestLength="2147482548"/>
```
Comment (gep13) [inline src/SimpleChocolateyServer/Web.config]: @ferventcoder if we are in doubt after changing this, we could do a quick test with a larger file.

Comment (ferventcoder) [inline src/SimpleChocolateyServer/Web.config]: The limit set on Chocolatey.org is crazy high, so this should be fine.

**Comment by ferventcoder** [comment]
This will be great!

**Pull request merged**